src/conversions/chrono.rs

```diff
@@ -0,0 +1,663 @@
+#![cfg(feature = "chrono")]
+
+//! Conversions to and from [chrono](https://docs.rs/chrono/)’s `Duration`,
+//! `NaiveDate`, `NaiveTime`, `DateTime<Tz>`, `FixedOffset`, and `Utc`.
+//!
+//! # Setup
+//!
+//! To use this feature, add this to your **`Cargo.toml`**:
+//!
+//! ```toml
+//! [dependencies]
+//! # change * to the latest versions
+//! pyo3 = { version = "*", features = ["chrono"] }
+//! chrono = "0.4"
+// workaround for `extended_key_value_attributes`: https://github.com/rust-lang/rust/issues/82768#issuecomment-803935643
+#![cfg_attr(docsrs, cfg_attr(docsrs, doc = concat!("pyo3 = { version = \"", env!("CARGO_PKG_VERSION"),  "\", features = [\"chrono\"] }")))]
+#![cfg_attr(
+    not(docsrs),
+    doc = "pyo3 = { version = \"*\", features = [\"chrono\"] }"
+)]
+//! ```
+//!
```

Comment:
I'd like (not require) an example showing why you might want to use the chrono feature. What does it let you do that you otherwise can't?

Reply:
Sure, I'll add a brief example and an entry to the doc page you linked

Reply:
wait, I think I broke the CI, pushing a fix

Reply:
Ok, example added and optional-feature-flags section updated.
I still have an issue with the CI as I mentioned in a previous comment, Clippy fails because it's running with the limited python api, but this integration needs it, how do you want to proceed here?